modules/core/mwaa/stack.py

```diff
@@ -192,6 +192,20 @@ def __init__(
                         f"arn:aws:dynamodb:{self.region}:{self.account}:table/addf-{deployment_name}-{module_name}*"
                     ],
                 ),
+                aws_iam.PolicyStatement(
+                    actions=["sagemaker:*"],
```

Comment:
This seems to be over permissive, `sagemaker:*` on "*"

Reply:
I restricted this to just Create/Describe/List Processing Jobs